modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -31,6 +31,7 @@ The following Technology Preview features are enabled by this feature set:
 ** Insights Operator. Enables the Insights Operator, which gathers {product-title} configuration data and sends it to Red Hat (`InsightsConfigAPI`).
 ** External cloud providers. Enables support for external cloud providers for clusters on vSphere, AWS, Azure, and GCP. Support for OpenStack is GA. (`ExternalCloudProvider`)
 ** Pod topology spread constraints. Enables the `matchLabelKeys` parameter for pod topology contraints. The parameter is list of pod label keys to select the pods over which spreading will be calculated (`MatchLabelKeysInPodTopologySpread`).
+** Pod security admission. Enables restricted enforcement for pod security admission. Instead of only logging a warning, pods are rejected if they violate pod security standards.
```

Comment:
```suggestion
** Pod security admission enforcement. Enables restricted enforcement for pod security admission. Instead of only logging a warning, pods are rejected if they violate pod security standards.
```
The name of the featuregate is `OpenShiftPodSecurityAdmission` (https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L136)

Reply:
I did see that, but didn't think it had to match exactly. But I will add the "OpenShift" in front as well, thanks!

Reply:
Updated. How does it look now?

Reply:
what I meant is that each of the above items mentions the name of the feature gate, only this last one does not